docker-compose.yaml

```diff
@@ -1,5 +1,13 @@
 version: '3'
 services:
+  pgadmin:
```

Comment:
so pgadmin here is not the postgres gui?

Reply:
it is the postgres gui